interfaces/builtin/browser_support_test.go

```diff
@@ -36,29 +36,32 @@ type BrowserSupportInterfaceSuite struct {
 	plug  *interfaces.Plug
 }
 
-var _ = Suite(&BrowserSupportInterfaceSuite{
-	iface: &builtin.BrowserSupportInterface{},
-	slot: &interfaces.Slot{
+const browserMockPlugSnapInfoYaml = `name: other
+version: 1.0
+plugs:
```

Comment:
This feels redundant. I agree that we don't need to spell it out. Let me try to push a small patch into this PR to simplify things.